Tasks/AzureCLI/azureclitask.ts

```diff
@@ -44,13 +54,18 @@ export class azureclitask {
             tl.mkdirP(cwd);
             tl.cd(cwd);
 
+            if (scriptType === "bash") {
+                tool.pathArg(scriptPath);
+            }
+            else {
+                tool = tl.createToolRunner(tl.which(scriptPath, true));
+            }
+
             var connectedServiceNameSelector = tl.getInput("connectedServiceNameSelector", true);
             this.loginAzure(connectedServiceNameSelector);
 
-            bash.pathArg(scriptPath);
-            bash.argString(args); // additional args should always call argString.  argString() parses quoted arg strings
-
-            await bash.exec({failOnStdErr: failOnStdErr});
+            tool.argString(args); // additional args should always call argString.  argString() parses quoted arg strings
```

Comment:
thanks. changed tool.argString to tool.line and 
tool.pathArg to tool.arg
@bryanmacfarlane